### PR TITLE
add ValidateAllUserBetaPrograms command and tests to profile client

### PR DESCRIPTION
**Pull request by sweetaz**

### CODE REVIEW

**Comment by GuillaumeBrousse** [comment]
Nice Work Aaron!

File: src/Profile/ProfileClient.php [hunk 1]
```diff
@@ -11,6 +11,9 @@
  *
  * @method \Serato\SwsSdk\Result getUser(array $args)
  * @method \Serato\SwsSdk\Result updateUser(array $args)
+ * @method \Serato\SwsSdk\Result getUserBetProgram(array $args)
```
Comment (GuillaumeBrousse) [inline src/Profile/ProfileClient.php]: You forgot a `A` to getUserBetaProgramm

Comment (sweetaz) [reply src/Profile/ProfileClient.php]: You added an extra m :D

Comment (GuillaumeBrousse) [reply src/Profile/ProfileClient.php]: ahah you saw 🍾 🤣

**Pull request merged**